docker/worker/worker.py

```diff
@@ -333,6 +333,10 @@ def _source_update(self, message):
     deleted = message.attributes['deleted'] == 'true'
 
     source_repo = osv.get_source_repository(source)
+    if source_repo is None:
+      logging.error('Failed to get source repository %s', source)
```

Comment:
To make this logged error more debuggable and actionable, it may be better to raise an exception to the caller, which will then be able to add the offending record's ID to the exception.

Reply:
yes - I also thought of adding ID to the error message so raising an exception is a good idea! I have made https://github.com/google/osv.dev/pull/2933 to address this.